### DISCUSSION TITLE
PR Merged into Master not in any Release

### DISCUSSION THREAD

**Discussion by TimoObst** [AvaloniaUI/Avalonia#14498] [General]
Hello all!

I am working on creating Custom Notifications, but was running into the issue the limitations of `WindowNotificationManager`. While looking into it a bit more, I noticed this PR: https://github.com/AvaloniaUI/Avalonia/pull/12178. It refactored the `WindowNotificationManager`, adding in the features and fixing the issues I was having. 

But when looking further into it, it has not been included in any release. It was merged into master on August 31st, and there have been 3 releases since then. Knowing that this is coming, I'd rather not develop a workaround now that I'll have to refactor later.

So my questions is... Why hasn't this been included in any release? Is a refactor like this not intended for a minor version release, and will I have to wait for 11.1.0?

**Comment by timunie** [comment]
yes, only bug fixes are backportet. You need to wait for 11.1 or use nightly builds https://github.com/AvaloniaUI/Avalonia/wiki/Using-nightly-build-feed

**Reply by TimoObst** [reply]
Thanks for the quick reply! I'll have a look at nightly builds
Is there some timeline of when we can expect releases?

**Reply by timunie** [reply]
Nope, it's done when it's done.
